User home page and google sign in added.

saurabhshukla689 opened the pull request:
Modified the styling of the user home page and added the functionality
of signing in through google plus

saurabhshukla689 committed:
User home page and google sign in added.

Modified the styling of the user home page and added the functionality
of signing in through google plus

Aniltilanthe commented on public/javascripts/angularApp.js:
```diff
@@ -133,4 +133,11 @@ function($scope, physicians, auth, users){
   };
+
+  $scope.goToThisDetails = function(ids,alignToTop){
+    //debugger;
```
1. Write a comment above the method explaining what it does & what should be passed as parameters
2. Remove all debuggers

Aniltilanthe commented on public/javascripts/controllers/AuthCtrl.js:
```diff
@@ -55,4 +55,27 @@ function($scope, $state, auth){
 //_______________________________________
+
+
+$scope.signinCallback =  function(authResult) {
```
Where is this method used ?
This method is repeated in PublicScript.js also. Right?

Aniltilanthe commented on public/javascripts/controllers/UserCtrl.js:
```diff
@@ -14,5 +14,10 @@ function($scope, users, auth){
   };
 
   loadGoogleCalendarEvents();
+  $scope.goToThisDetails=function(alignToTop){
```
1. Is this method same as the one used in angularApp.js
   Which one is used ?
   Remove the one which is not used.
2. Add a comment above the method 
   - explain what it does
   - what are the parameters passed to it & what should be there expected value
     3, Remove the debugger

Aniltilanthe commented on public/stylesheets/style2.css:
```diff
@@ -0,0 +1,93 @@
+body {
```
Do we need two separate files for CSS styles ?
Can we add this in the other styles file ?

Aniltilanthe commented on public/stylesheets/vertical.css:
```diff
@@ -0,0 +1,34 @@
+@media (min-width: 768px) {
```
Do we need one more styles file ? 
Application loading slows down with more files.

Aniltilanthe commented on views/index.ejs:
```diff
@@ -12,11 +11,12 @@
   <!-- libraries -->
   <script src="/javascripts/libraries/ui-bootstrap-tpls-0.13.0.min.js"></script>
-
+<script src="https://apis.google.com/js/client:platform.js" async defer> </script>
```
This file is being requested twice ?
See the green line above this.
Do we need to increase the application load time by making multiple requests ?

Aniltilanthe commented on views/index.ejs:
```diff
@@ -67,79 +68,128 @@
   <div class="row">
-    <div class="col-md-12 m-body">
-
+    </div>
```
Why is this closed here?

Aniltilanthe commented on views/index.ejs:
```diff
@@ -67,79 +68,128 @@
+  
+        
+  <div class="row">
+    <div class="col-sm-2 "   style="background-color:#d2b481;position:fixed">
```
Add a class here with a name & put this property into that class.
Add the class here

Aniltilanthe commented on views/index.ejs:
```diff
@@ -67,79 +68,128 @@
+<br><br><br><br><br><br><br><br><br><br><br><br>
+
+<div id="doctors">
+<div class="jumbotron">
```
Add Space . Formatting is very important so that others can understand faster. :)

Aniltilanthe commented on views/index.ejs:
```diff
@@ -67,79 +68,128 @@
+<!--_________________________________________________________________________-->
+
+  <script type="text/ng-template" id="/home.html">
+  <link rel="stylesheet"  type="text/css" href="style2.css">
```
If this needed ?

saurabhshukla689 replied:
<div>
>          <!-- Insert html view templates here -->
>            <ui-view></ui-view>

This code is needed because the home page would not show up without
this custom directive.

However,I merged all the CSS in one file


On Thu, Jun 11, 2015 at 2:36 PM, Aniltilanthe <notifications@github.com>
wrote:

> In views/index.ejs
> <https://github.com/Aniltilanthe/MasterApp/pull/2#discussion_r32201903>:
>
> >        <div>
> >          <!-- Insert html view templates here -->
> >            <ui-view></ui-view>
> >
> > +
> > +    </div>

> > +
> > +
> > +<!--_________________________________________________________________________-->
> > +
> > -  <script type="text/ng-template" id="/home.html">
> > -  <link rel="stylesheet"  type="text/css" href="style2.css">
> 
> If this needed ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Aniltilanthe/MasterApp/pull/2/files#r32201903.

Aniltilanthe commented on views/index.ejs:
```diff
@@ -67,79 +68,128 @@
+      <br><button type="button" class="btn btn-warning btn-sm" style="margin-left:45px ">Logout</button>
+<br><br><br><br>
+</div>
+<div class="sidebar-nav" style="background-color:#d2b481;padding:0px 0px 0px 0px;margin:0px 0px 0px 0px;border:1px #d2b481 ">
```
Set this as a class.  Everywhere do the same.

Aniltilanthe commented on views/index.ejs:
```diff
@@ -67,79 +68,128 @@
+  <link rel="stylesheet"  type="text/css" href="style2.css">
+  <link rel="stylesheet"  type="text/css" href="vertical.css">
+
+<div class="container-fluid">
```
This is to be put into the userHome.html template. 
There is no change in this template for now.

saurabhshukla689 committed:
Modified

Added the necessary comments,properly formatted and put all the CSS in
one file

Aniltilanthe commented on GlobalData.txt:
```diff
@@ -18,4 +18,8 @@ Referrers: http://localhost:3000/
 
-_______________________________________________________________________
+____________________________________________________________________
+Client ID for google sign in button
```
Why have you pushed this ?
This is not needed right ? We have used the first one which is existing already.
Can you write all the changes in a paper & then do them each. So that you do not forget anything.

Aniltilanthe commented on public/javascripts/controllers/AuthCtrl.js:
```diff
@@ -55,4 +55,10 @@ function($scope, $state, auth){
 
   $scope.speciality = "Psychiatry";
 //_______________________________________
+
```
If there is no change then do not push a file.

Aniltilanthe commented on public/javascripts/script/publicScript.js:
```diff
@@ -179,3 +180,48 @@ function createGoogleCalendarEvent(userEmail, physicianEmail, physicianUsername,
+
+
+signinCallback =  function(authResult) {
+    console.log("HemllO");
```
Remove this console & the debugger.

Aniltilanthe commented on public/javascripts/script/publicScript.js:
```diff
@@ -179,3 +180,48 @@ function createGoogleCalendarEvent(userEmail, physicianEmail, physicianUsername,
+      console.log('Image URL: ' + resp.image.url);
+      console.log('Profile URL: ' + resp.url);
+    });
+
```
Do you need all these new line characters ?

Aniltilanthe commented on public/stylesheets/style.css:
```diff
@@ -1,3 +1,70 @@
+}
+}
+
+
```
Do you need so many New Line characters ?

Aniltilanthe commented on views/index.ejs:
```diff
@@ -12,11 +11,11 @@
   <!-- libraries -->
   <script src="/javascripts/libraries/ui-bootstrap-tpls-0.13.0.min.js"></script>
-
+<script src="https://apis.google.com/js/client:platform.js" async defer> </script>
```
This is already present na ?  
Why are you requesting for the same file again ? This is being loaded.

saurabhshukla689 committed:
Changes

saurabhshukla689 commented:
No worries,I will fix it all tonight
Thanks

On Fri, Jun 12, 2015 at 6:37 PM, Aniltilanthe notifications@github.com
wrote:

> Merged #2 https://github.com/Aniltilanthe/MasterApp/pull/2.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Aniltilanthe/MasterApp/pull/2#event-329532418.
